packages/salesforcedx-vscode-core/src/conflict/conflictUtils.ts

```diff
@@ -125,7 +111,7 @@ export const handleConflictsWithUI = async <T>(
 /**
  * Resolves conflict metadata component references to actual file paths
  */
-const resolveConflictFilePaths = async (conflicts: any[]): Promise<string[]> => {
+const resolveConflictFilePaths = async (conflicts: Conflict[]): Promise<string[]> => {
   const conflictFilePaths = conflicts.map(
     conflict => conflict.filePath ?? conflict.fullName ?? conflict.name ?? 'unknown'
```

Comment:
now you get type support on Conflict!.  You'll see the filePath doesn't exist, nor does fullName.  

You can get an array of `filenames` and maybe flatten that to get your file path list?

This is important because you can get conflicts in things that have multiple files (think bundles, or ApexClass with its meta-xml)

STL has already done the hard work of trying to match server names with local files.

If the goal is to get file paths, then this won't work as is.